### PR TITLE
Document the GraphQL Usage API

### PR DESCRIPTION
**Pull request by jameshill**
## Usage API via GraphQL

We've recently added the ability to query Usage data via the GraphQL API. 

## Internal Context
- [Project Page](https://3.basecamp.com/3453178/buckets/29554282/messages/6006048651)
- [Coda](https://coda.io/d/Buildkite-Roadmap_dUWyuyLHsQw/Marketing-Release-Train_su-L0#June-Release_tuTsj/r1&view=modal)

### CODE REVIEW

**Comment by mbelton-buildkite** [comment]
(FYI, I've seen this but not reviewed it because it's in draft. Mention me in a comment when you're ready for review 😊)

**Comment by jameshill** [comment]
Hey @mbelton-buildkite this is ready for review, no change to before... I was just checking on the auto-generated (schema) documentation.

File: pages/apis/graphql/graphql_cookbook.md [hunk 1]
```diff
@@ -650,6 +650,52 @@ mutation UpdateSessionIPAddressPinning {
...
+
+```graphql
+query Usage {
+  organization(slug: "organization-slug") {
+    id
+    name
+    usage(
+      aggregatedOnFrom: "2023-04-01"
+      aggregatedOnTo: "2023-05-01"
+      resource:[JOB_MINUTES, TEST_EXECUTIONS]
```
Comment (mbelton-buildkite) [inline pages/apis/graphql/graphql_cookbook.md:665]: Style:

```suggestion
      resource: [JOB_MINUTES, TEST_EXECUTIONS]
```

File: pages/apis/graphql/graphql_cookbook.md [hunk 2]
```diff
@@ -650,6 +650,52 @@ mutation UpdateSessionIPAddressPinning {
...
+          __typename ... on JobMinutesUsage {
+            aggregatedOn
+            seconds
+            pipeline {
+              name
+              id
+            }
+          }
+        }
+
```
Comment (mbelton-buildkite) [inline pages/apis/graphql/graphql_cookbook.md:678]: Style:

```suggestion
```

File: pages/apis/graphql/graphql_cookbook.md [hunk 3]
```diff
@@ -650,6 +650,52 @@ mutation UpdateSessionIPAddressPinning {
...
+        node {
+          __typename ... on TestExecutionsUsage {
+            Time:aggregatedOn
```
Comment (mbelton-buildkite) [inline pages/apis/graphql/graphql_cookbook.md:681]: Style:

```suggestion
            Time: aggregatedOn
```

File: pages/pipelines/job_minutes.md [hunk 4]
```diff
@@ -21,6 +21,8 @@ The [Usage page](https://buildkite.com/organizations/~/usage) is available on ev
 
 The [Job minutes usage page](https://buildkite.com/organizations/~/usage/job_minutes) graphs the total job minute usage over the organization's billing periods. It includes a breakdown of usage by pipeline and a CSV download of usage over the period.
 
+Your organization's usage is also accessible via the [GraphQL API](/docs/apis/graphql/graphql-cookbook#organizations-query-the-usage-api).
```
Comment (mbelton-buildkite) [inline pages/pipelines/job_minutes.md:24]: Style nitpick: "via" can be ambiguous so I generally avoid it in docs.

```suggestion
Your organization's usage is also accessible in the [GraphQL API](/docs/apis/graphql/graphql-cookbook#organizations-query-the-usage-api).
```

File: pages/test_analytics/test_executions.md [hunk 5]
```diff
@@ -10,3 +10,5 @@ You can find the test execution details for a run at the bottom of the run page,
 The [Usage page](https://buildkite.com/organizations/~/usage) is available on every Buildkite plan and shows a breakdown of job minutes and test executions for your organization.
 
 The [test executions usage page](https://buildkite.com/organizations/~/usage/test_executions) graphs the total executions over the organization's billing periods. It includes a breakdown of usage by suite and a CSV download of usage over the period.
+
+Your organization's usage is also accessible via the [GraphQL API](/docs/apis/graphql/graphql-cookbook#organizations-query-the-usage-api).
```
Comment (mbelton-buildkite) [inline pages/test_analytics/test_executions.md:14]: Style nitpick: "via" can be ambiguous so I generally avoid it in docs.

```suggestion
Your organization's usage is also accessible in the [GraphQL API](/docs/apis/graphql/graphql-cookbook#organizations-query-the-usage-api).
```

**Review by mbelton-buildkite — COMMENTED** [review]
Just a few style tweaks.

I remember this was added by support to provide some guidance on usage: [Get number of builds between two dates](https://2220--bk-docs-preview.netlify.app/docs/apis/graphql/graphql-cookbook#builds-get-number-of-builds-between-two-dates). Do you think it should be removed now? All good to just keep it if it's not an easy answer.

**Review by mbelton-buildkite — APPROVED** [review]

**Comment by jameshill** [comment]
> I remember this was added by support to provide some guidance on usage: [Get number of builds between two dates](https://2220--bk-docs-preview.netlify.app/docs/apis/graphql/graphql-cookbook#builds-get-number-of-builds-between-two-dates). Do you think it should be removed now? All good to just keep it if it's not an easy answer.

Good idea, yeah I reckon we can drop this in a new PR.

**Pull request merged**